README.md

```diff
@@ -453,12 +459,6 @@ export class CrewMembersComponent extends NgxSubFormRemapComponent<CrewMember[],
     switch (key) {
       // note: the following string is type safe based on your form properties!
       case 'crewMembers':
-        return new FormControl(value, [Validators.required]);
```

Comment:
isn't this invalid typescript now? Should use embedme :)

Reply:
Whew. So bad news is... I didn't touch that part of the readme, just ran embedme :D

Reply:
I needed to update the line numbers. Done